<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormHighPassFilterSettingDialog</class>
 <widget class="QWidget" name="FormHighPassFilterSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ядро</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radio_3x3">
        <property name="text">
         <string>3x3</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_5x5">
        <property name="text">
         <string>5x5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_7x7">
        <property name="text">
         <string>7x7</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
